### PR TITLE
Register Subdomain: zuhair

### PR DESCRIPTION
**Pull request by ZuhairTarif**
Added the domain: zuhair

### CODE REVIEW

**Review by wdhdev — APPROVED** [review]

**Comment by ZuhairTarif** [comment]
Hey just wanted to know about my review processes

**Comment by WilliamDavidHarrison** [comment]
> Hey just wanted to know about my review processes

You just need to wait for a maintainer to merge it, I'm only a helper so I can't merge it.

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**
